### PR TITLE
mbw 1.5 (new formula)

### PR DESCRIPTION
**Pull request by Willian-Zhang**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by Willian-Zhang** [comment]
Cannot pass `brew audit --new mbw`
```
mbw:
  * GitHub fork (not canonical repository)
Error: 1 problem in 1 formula detected
```
as It is indeed a fork, with some modifications and not previously been listed on homebrew.

**Comment by SMillerDev** [comment]
> as It is indeed a fork, with some modifications and not previously been listed on homebrew.

Why not contribute those back upstream?

**Comment by Willian-Zhang** [comment]
> > as It is indeed a fork, with some modifications and not previously been listed on homebrew.
> 
> Why not contribute those back upstream?

The upstream seems not to be actively maintained. Also I'm planing on adding more features to the original one, it could be much different to the original tool at the end.
Nevertheless I'm on reaching. Hope to see this happen soon.

**Comment by Willian-Zhang** [comment]
If not, should I be changing the name of this contributed formula or if it's okey to release under the same?  @SMillerDev

**Comment by raas** [comment]
I'm the original author of mbw, and I'm happy to add you as a maintainer there to avoid forking.

**Comment by Willian-Zhang** [comment]
Things are sorted out, pushing new version 1.5

**Comment by Willian-Zhang** [comment]
@SMillerDev ready to merge

File: Formula/mbw.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    pipe_output("#{bin}/mbw", 0)
```
Comment (SMillerDev) [inline Formula/mbw.rb:17]: Can we check some output here like
```suggestion
    assert_match "something", pipe_output("#{bin}/mbw", 0)
```

**Review by SMillerDev — APPROVED** [review]
Thanks @Willian-Zhang and @raas  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**